libs/core-ui/src/lib/Highchart/HighchartWrapper.tsx

```diff
@@ -3,10 +3,10 @@
 
 import { ITheme } from "@fluentui/react";
 import _ from "lodash";
-import { mergeStyles } from "office-ui-fabric-react";
+import { getTheme, mergeStyles } from "office-ui-fabric-react";
```

Comment:
why do we need both @fluentui/react and office-ui-fabric-react, I thought they were the same package?  Just for my understanding.

Reply:
They are, we are in the process of migrating from fabric ui to fluentui, eventually fluentui will be the one always imported from.